controllers/machine_controller.go

```diff
@@ -289,6 +308,39 @@ func (r *MachineReconciler) tryLock(machine *naglfarv1.Machine, dockerClient *do
 	return
 }
 
+func (r *MachineReconciler) createUploadDaemon(machine *naglfarv1.Machine, dockerClient *dockerutil.Client) (uploadPort int, err error) {
+	cfg, hostCfg := container.UploadDaemonCfg(string(machine.UID))
+	stat, err := dockerClient.Run(container.UploadDaemon, container.UploadDaemonImage, dockerutil.RunOptions{
+		Config:     cfg,
+		HostConfig: hostCfg,
+	})
+
+	if err != nil {
+		return
+	}
+
+	if stat.Config == nil || stat.Config.Labels == nil {
+		err = fmt.Errorf("invalid occupy container: %s", stat.Name)
+	}
+
+	uid, ok := stat.Config.Labels[container.UploadLabel]
+	if !ok {
+		err = fmt.Errorf("container %s has no label `%s`", stat.Name, container.UploadLabel)
+		return
+	}
+
+	if uid != string(machine.UID) {
+		err = fmt.Errorf("machine(%s) occupied by other naglfar system: UID(%s)", machine.Name, machine.UID)
```

Comment:
Why do you check the occupation? it has been checked on createUploadDaemon.

Reply:
![image](https://user-images.githubusercontent.com/59600278/105655545-3392e800-5efb-11eb-9e1e-c1517d7bff61.png)
There is no check in createUploadDaemon. I refer to Xixi code.

Reply:
The cursor you point to is the check logic, it has been check again and again.

Reply:
![image](https://user-images.githubusercontent.com/59600278/105665380-e40be680-5f11-11eb-9b7a-c2d7b8d5a42f.png)
![image](https://user-images.githubusercontent.com/59600278/105665469-1f0e1a00-5f12-11eb-96b0-25da80cebae6.png)
This code is in createdaemon. Do you mean I should move this part of the code to reconcilestating

Reply:
done